public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -1,34 +1,35 @@
-include ../_util-fns
-
-<!-- http://plnkr.co/edit/x9JYbC -->
-
+block includes
+  include ../_util-fns
+  - var _iterableUrl = 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols';
+  - var _boolean = 'truthy/falsey';
+  
 :marked
   We typically display data in Angular by binding controls in an HTML template
   to properties of an Angular component.
 
   In this chapter, we'll create a component with a list of heroes. Each hero has a name.
   We'll display the list of hero names and
-  conditionally show a selected hero in a detail area below the list.
+  conditionally show a message below the list.
 
   The final UI looks like this:
 
 figure.image-display
   img(src="/resources/images/devguide/displaying-data/final.png" alt="Final UI")
-  
+
 :marked
   # Table Of Contents
-  
+
   * [Showing component properties with interpolation](#interpolation)
-  * [Showing an array property with NgFor](#ngFor)
+  * [Showing !{_an} !{_array} property with NgFor](#ngFor)
   * [Conditional display with NgIf](#ngIf)
-  
+  <li if-docs="ts">only ts</li>
```

Comment:
That was just for a test. It should not have been left in (no sure how it slipped by). Thanks for catching it!

Fixed.